File: spring-backend/src/main/java/com/petcare/PetcareApplication.java
package com.petcare;

/*
 * Comentario de modulo PetCare:
 * Clase Java del proyecto PetCare. Mantiene compatibilidad con partes del backend que aun no estan en Kotlin.
 */

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;

@SpringBootApplication
public class PetcareApplication {
    public static void main(String[] args) {
        SpringApplication.run(PetcareApplication.class, args);
    }
}

File: src/main/kotlin/com/petcare/PetcareApplication.kt
package com.petcare

/*
 * Comentario de modulo PetCare:
 * Archivo del proyecto PetCare. Mantiene una parte especifica de la app y debe conservarse simple de seguir.
 */

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication
import org.springframework.scheduling.annotation.EnableScheduling

@SpringBootApplication
@EnableScheduling
class PetcareApplication

fun main(args: Array<String>) {
    runApplication<PetcareApplication>(*args)
}
